src/components/editors/GroupsEditor/components/GroupEditor/components/LinkEditor/LinkEditor.tsx

```diff
@@ -119,6 +133,11 @@ export const linkTypeOptionsInDropdown = [
     label: 'Timepicker',
     description: 'Set time range',
   },
+  {
```

Comment:
It seems that 2 these lists are crossed. Can we remove a duplication? Like creating one map with all available options and use its items in these lists?